.travis.yml

```diff
@@ -1,24 +1,80 @@
-sudo: required
-dist: trusty
+os:
+ - linux
+ - osx
+dist: xenial
 language: java
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y graphviz
+jdk:
+  - openjdk8
```

Comment:
@Vuksaa, kill OpenJDK8. Let's make sure this works correctly on JDK11 and above.